packages/server/__tests__/email/email.test.js

```diff
@@ -19,23 +19,47 @@ const {
 } = process.env;
 
 const testEmail = {
-    toAddress: TEST_EMAIL_RECIPIENT || 'test@gmail.com',
+    toAddress: TEST_EMAIL_RECIPIENT || 'nobody@example.com',
     subject: 'Test email',
     body: 'This is a test email.',
 };
 
 describe('Email module', () => {
+    function restoreEnvironmentVariables() {
+        process.env.TEST_EMAIL_RECIPIENT = TEST_EMAIL_RECIPIENT;
+        process.env.AWS_ACCESS_KEY_ID = AWS_ACCESS_KEY_ID;
+        process.env.AWS_SECRET_ACCESS_KEY = AWS_SECRET_ACCESS_KEY;
+        process.env.SES_REGION = SES_REGION;
+        process.env.NOTIFICATIONS_EMAIL = NOTIFICATIONS_EMAIL;
+        process.env.NODEMAILER_HOST = NODEMAILER_HOST;
+        process.env.NODEMAILER_PORT = NODEMAILER_PORT;
+        process.env.NODEMAILER_EMAIL = NODEMAILER_EMAIL;
+        process.env.NODEMAILER_EMAIL_PW = NODEMAILER_EMAIL_PW;
+    }
+
+    function clearNodemailerEnvironmentVariables() {
+        delete process.env.NODEMAILER_HOST;
+        delete process.env.NODEMAILER_PORT;
+        delete process.env.NODEMAILER_EMAIL;
+        delete process.env.NODEMAILER_EMAIL_PW;
+    }
+
+    function clearSESEnvironmentVariables() {
+        delete process.env.AWS_ACCESS_KEY_ID;
+        delete process.env.AWS_SECRET_ACCESS_KEY;
+        delete process.env.SES_REGION;
+        delete process.env.NOTIFICATIONS_EMAIL;
+    }
+
+    afterEach(() => {
+        restoreEnvironmentVariables();
```

Comment:
Was there some specific bug you ran into where this was needed?

Reply:
No; I did this while debugging the original failure because I thought the issue might be related to side effects from previous tests. It turned out not to be, but this felt cleaner anyway -- before I think we were depending on the order of tests executing since some of the setup/cleanup was in `it(...)` blocks.